script/GenerateDefaultOperatorsModule.purs

```diff
@@ -31,7 +31,7 @@ main = do
 
   writeTextFile UTF8 (Path.concat [ tmpPath, "spago.dhall" ]) defaultSpagoDhall
   writeTextFile UTF8 (Path.concat [ tmpPath, "package.json" ]) defaultPackageJson
-  _ <- ChildProcess.execSync "npm install" opts
+  -- _ <- ChildProcess.execSync "npm install" opts
```

Comment:
I had to disable this because I run on NixOS and npm installing PureScript doesn't work there. I'm not sure it's technically necessary to do this, if the developer already has spago and purescript in their paths.

```suggestion
  _ <- ChildProcess.execSync "npm install" opts
```

Reply:
The reason I added this was because I was building 0.15 artifacts with a version of the formatter compiled with 0.14. If the version of the compiler you are running doesn't match version of the compiler you are formatting, you're hosed because `spago install` won't work :/.

Reply:
Yea, I know NixOS is a weird system to even attempt to support, so no qualms from me. Unless you wanted to use Nix shells, which I would assume you don't.